ios/Framework/Core/WebContent/DisplayScreenlet/WebContentDisplayScreenlet.swift

```diff
@@ -123,12 +123,11 @@ open class WebContentDisplayScreenlet: BaseScreenlet {
 				self.webContentDisplayViewModel?.customCssFile = self.customCssFile
 				self.webContentDisplayViewModel?.htmlContent = modifiedHtml ?? resultHtml
 
-                let onUrlClicked: ((WebContentDisplayScreenlet, String) -> Bool)? =
-                    self.webContentDisplayDelegate?.screenlet
+				let onUrlClicked: ((WebContentDisplayScreenlet, String) -> Bool)? =
+					self.webContentDisplayDelegate?.screenlet
 
-                self.webContentDisplayViewModel?.onUrlClicked = { url in
-                    return self.onUrlClicked(url)
-                }
+				self.webContentDisplayViewModel?.onUrlClicked = self.onUrlClicked(_:)
+				
```

Comment:
here you can use weakify(owner: self, WebContentDisplayScreenlet.onUrlClicked) to avoid creating a reference cycle